plugin/file_key_management/parser.cc

```diff
@@ -335,7 +335,7 @@ char* Parser::read_and_decrypt_file(const char *secret)
                    ME_ERROR_LOG , filename, errno);
     goto err2;
   }
-  fprintf(stderr, "Read from %s , filesize: %ldB,max key file size :%d B\n " , filename,(size_t)file_size,MAX_KEY_FILE_SIZE);
+  fprintf(stderr, "Read from %s , filesize: %ldB,max key file size :%d B\n " , filename,(ulong)file_size,MAX_KEY_FILE_SIZE);
```

Comment:
note a `ssize_t`  uses format `%zd` and then won't need a cast regardless of platform.

Reply:
Thanks a lot . And here I use ```fprintf`` .Is there other better log function instead?

Reply:
`my_printf_error` seems the best I could find.

Reply:
Let me think, But this maybe is not an error . This real read bytes is logged to tell user and diagnose when user provide a large key file.

Reply:
Any comments would be great on codes .

Reply:
> `my_printf_error` seems the best I could find.

Haha, I understand what you mean and it works . Error log file  is maraidb  log file by design.